tree/dataframe/inc/ROOT/RDFGraphUtils.hxx

```diff
@@ -0,0 +1,132 @@
+// Author: Enrico Guiraud, Danilo Piparo, CERN, Massimo Tumolo Politecnico di Torino 08/2018
+
+/*************************************************************************
+ * Copyright (C) 1995-2016, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_GRAPHUTILS
+#define ROOT_GRAPHUTILS
+
+#include <string>
+#include <sstream>
+#include <vector>
+#include <map>
+#include <memory>
+#include <ROOT/RDataFrame.hxx>
+#include <ROOT/RDFInterface.hxx>
+#include <ROOT/GraphNode.hxx>
+
+#include <iostream>
+
+namespace ROOT {
+namespace Detail {
+namespace RDF {
+class RCustomColumnBase;
+class RFilterBase;
+class RRangeBase;
+} // namespace RDF
+} // namespace Detail
+
+namespace Internal {
+namespace RDF {
+std::shared_ptr<GraphNode>
```

Comment:
Why not adding a namespace `GraphDrawing`?

Reply:
Ok, to avoid polluting the code I had to use an "using namespace RDFGraphDrawing = ROOT::Internal::RDF::GraphDrawing" inside RDF Internal and RDF Detail. I hope that's not a problem.

Reply:
We have quite a few of these nice shortcuts. Not at all.